pkg/database/plugin.go

```diff
@@ -706,3 +719,12 @@ var BlobQueryFactory = &queryFields{
 	"payloadref": &StringField{},
 	"created":    &TimeField{},
 }
+
+// TokenPoolQueryFactory filter fields for token pools
+var TokenPoolQueryFactory = &queryFields{
+	"id":          &UUIDField{},
+	"type":        &StringField{},
+	"namespace":   &StringField{},
+	"name":        &StringField{},
+	"protocol_id": &StringField{},
```

Comment:
We don't use underscores in our api field names (we use camel case - and here is case insensitive) - only the database schemas use underscores.
You should have a mapping in the SQL library to map the name
```suggestion
	"protocolid": &StringField{},
```

Reply:
Alright, so the convention is:
- ProtocolID in Go structs
- protocolid in JSON bodies
- protocol_id in database columns